azure-quantum/azure/quantum/problem_pb2.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suggest to move this file to a subpackage, e.g. azure/quantum/serialization/protobuf.py